### ISSUE TITLE
The plugin does not work with webpack

### ISSUE THREAD

**Issue by ignaciolarranaga** [NativeScript/nativescript-imagepicker#43]
It raises this error when trying to select:

`Error: Failed to load Page from entry.moduleName: ./tns_modules/nativescript-imagepicker/albums`

But it works correctly if not run with --bundle.
I think it is because webpack did not detect the navigations so not inspecting the other components.

**Comment by ignaciolarranaga** [comment]
Here seems to be the solution, but I'm still not able to make it work: https://github.com/NativeScript/nativescript-dev-webpack/issues/6

**Comment by ignaciolarranaga** [comment]
Progressed partially by using this for example:

```
if (global.TNS_WEBPACK) {
    require("globals");
    global.registerModule("tns_modules/nativescript-imagepicker/albums", () => require("./albums.ios.js"));
}
```

but now the problem is the xml. Even if added into the bundle the frame-common#pageFromBuilder() does not recognized it, so it doesn't work.

**Comment by ignaciolarranaga** [comment]
Just to comment, the JS default project does not work: https://github.com/NativeScript/NativeScript/issues/3066, but the Angular version (which is my case), does actually work with Webpack.

**Comment by ignaciolarranaga** [comment]
I'll replace the xml views with JS implementations through createPage, I think it might work.

**Comment by ignaciolarranaga** [comment]
Adding the PR #44 with a tentative solution. It does work, but may be you guys can find a better one.

**Comment by ignaciolarranaga** [comment]
It should be fixed PR #44
